### PR TITLE
app-text/zathura: add 0.5.4

### PR DESCRIPTION
**Pull request by turret-dev**
zathura 0.5.3 changes some things in the tests meson file, requiring a separate patch to be rebased from master.
the dependencies for 0.5.3 have been both increased and hardened (i.e., requirement on >=dev-libs/girara-0.4.1, new requirement on json-glib)

finally, upstream maintains an API/ABI versioning system separate from the package version itself, and has bumped both API and ABI without a minor/major version number bump, thus i've opted to move the subslots to be independent of the actual version number

needs #34186 merged before merging

### CODE REVIEW

File: app-text/zathura/zathura-0.5.3.ebuild [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+	seccomp? ( sys-libs/libseccomp )
+	sqlite? ( >=dev-db/sqlite-3.6.23:3 )
+	synctex? ( app-text/texlive-core )
+"
+RDEPEND="${DEPEND}"
+BDEPEND="
+	test? (
+		dev-libs/appstream-glib
+		dev-libs/check
+		x11-base/xorg-server[xvfb]
```
Comment (sevz17) [inline app-text/zathura/zathura-0.5.3.ebuild:46]: virtualx.eclass already add this

Another thing, if the tests are run under xvfb, it also needs x11-libs/gtk+[X]

Comment (turret-dev) [reply app-text/zathura/zathura-0.5.3.ebuild]: thank you for your review!
don't remember why I added that requirement in my eapi bump, i remember def. getting portage bugging me about xvfb, but can't remember exacts. :p
thanks for the recommendation on x11-libs/gtk+[X], i've added that

File: app-text/zathura/zathura-0.5.3.ebuild [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+RESTRICT="!test? ( test )"
+
+DEPEND="
+	>=dev-libs/girara-0.4.1
+	>=dev-libs/glib-2.50:2
+	dev-libs/json-glib
+	sys-apps/file
+	>=sys-devel/gettext-0.19.8
+	x11-libs/cairo
+	>=x11-libs/gtk+-3.22:3[X]
```
Comment (sevz17) [inline app-text/zathura/zathura-0.5.3.ebuild:36]: What I meant was 
```
DEPEND="
	>=x11-libs/gtk+-3.22:3
	...
	test? ( x11-libs/gtk+[X] )
"
```

Comment (turret-dev) [reply app-text/zathura/zathura-0.5.3.ebuild]: ic. i'll do what you've done over at girara as that does make the most sense at the moment and coincides with the devmanual.

**Comment by thesamesam** [comment]
Please fix CI.

**Comment by sevz17** [comment]
@thesamesam this depends on #34186

File: app-text/zathura/zathura-0.5.4.ebuild [hunk 3]
```diff
@@ -13,38 +13,50 @@ if [[ ${PV} == *9999 ]]; then
...
+		https://github.com/pwmt/zathura/archive/${PV}.tar.gz -> ${P}.tar.gz
+		https://cdn.turret.cyou/~turret/distfiles/${PN}/${P}-manpages.tar.xz
+	"
+	KEYWORDS="~amd64 ~arm ~riscv ~x86 ~amd64-linux ~x86-linux"
 fi
 
 LICENSE="ZLIB"
-SLOT="0"
-IUSE="doc +magic seccomp sqlite synctex test"
+SLOT="0/5.6"
```
Comment (thesamesam) [inline app-text/zathura/zathura-0.5.4.ebuild:24]: In a followup, consider adding the details from the commit message as a comment above `SLOT` too.

**Comment by thesamesam** [comment]
Thanks! I've updated the commit messages to not have a verison before `:`.

**Comment by thesamesam** [comment]
Please also look at:
```
$ pkgcheck scan
app-text/zathura
  RedundantVersion: version 0.5.2-r5: slot(0) keywords are overshadowed by version: 0.5.4
  StableRequest: version 0.5.2-r5: slot(0) no change in 145 days for unstable keywords: [ ~amd64, ~arm, ~x86 ]
  MissingUseDepDefault: version 0.5.4: DEPEND="x11-libs/gtk+[X]": USE flag 'X' missing from package: [ x11-libs/gtk+-2.24.33-r3 ]
  MissingUseDepDefault: version 9999: DEPEND="x11-libs/gtk+[X]": USE flag 'X' missing from package: [ x11-libs/gtk+-2.24.33-r3 ]
```

**Comment by thesamesam** [comment]
(actually, I'll just fix it)

**Comment by turret-dev** [comment]
thanks sam! (you spoil me)

**Pull request closed without merging**